cmd/prchecker/main_test.go

```diff
@@ -0,0 +1,232 @@
+package main
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/google/go-github/github"
+)
+
+func TestStringAllowsModuleSpan(t *testing.T) {
+	var tests = []struct {
+		body          string
+		matchExpected bool
+	}{
+		{"foo", false},
+		{"don't ALLOW_MODULE_SPAN", false},
+		{"ALLOW_MODULE_SPAN", true},       // PR body is just exception
+		{"> ALLOW_MODULE_SPAN", true},     // support quoting exception
+		{"\t ALLOW_MODULE_SPAN\t ", true}, // support whitespace padding
+		{"ALLOW_MODULE_SPAN\nat start of file", true},
+		{"at end of file\nALLOW_MODULE_SPAN", true},
+	}
+
+	for _, tt := range tests {
```

Comment:
A better pattern here is:
```
for i := range tests {
  test := tests[i]
  t.Run(test.name, func(t *testing.T) {
...
```
It doesn't seem like this code is covered by a linter, but it will complain about using iteration variables within an inner function.

Reply:
I had tried to match syntax to the example's [here](https://gobyexample.com/testing). Is there a reason the current syntax is problematic? I understand that there is potential to shadow variables in for loops but that should not arise here since execution of each tests is hermetic. Am I overlooking something?

Reply:
Using iterator variables inside an inner function is usually something a go linter will complain about. It is safe in this case though, so this is ok.